source/backend/dal/Repositories/PropertyRepository.cs

```diff
@@ -256,6 +256,47 @@ public PimsProperty GetByPin(int pin, bool includeRetired = false)
                 .FirstOrDefault(p => p.Pin == pin) ?? throw new KeyNotFoundException();
         }
 
+        /// <summary>
+        /// Get the properties that dont hace PIN or PID but have a plan#.
+        /// </summary>
+        /// <param name="plan"></param>
+        /// <returns></returns>
+        public PimsProperty GetWithOnlyPlan(string plan, bool includeRetired = false)
+        {
+            this.User.ThrowIfNotAllAuthorized(Permissions.PropertyView);
+
+            var query = Context.PimsProperties.AsNoTracking();
+
+            if (!includeRetired)
+            {
+                query = query.Where(r => !r.IsRetired.HasValue || (r.IsRetired.HasValue && !r.IsRetired.Value));
+            }
+
+            return query.Include(p => p.DistrictCodeNavigation)
+                .Include(p => p.RegionCodeNavigation)
+                .Include(p => p.PropertyTypeCodeNavigation)
+                .Include(p => p.PropertyStatusTypeCodeNavigation)
+                .Include(p => p.PropertyDataSourceTypeCodeNavigation)
+                .Include(p => p.PimsPropPropAnomalyTyps)
+                    .ThenInclude(t => t.PropertyAnomalyTypeCodeNavigation)
+                .Include(p => p.PimsPropPropRoadTyps)
+                    .ThenInclude(t => t.PropertyRoadTypeCodeNavigation)
+                .Include(p => p.PimsPropPropTenureTyps)
+                    .ThenInclude(t => t.PropertyTenureTypeCodeNavigation)
+                .Include(p => p.PropertyAreaUnitTypeCodeNavigation)
+                .Include(p => p.VolumetricTypeCodeNavigation)
+                .Include(p => p.VolumeUnitTypeCodeNavigation)
+                .Include(p => p.Address)
+                    .ThenInclude(a => a.RegionCodeNavigation)
+                .Include(p => p.Address)
+                    .ThenInclude(a => a.DistrictCodeNavigation)
+                .Include(p => p.Address)
+                    .ThenInclude(a => a.ProvinceState)
+                .Include(p => p.Address)
+                    .ThenInclude(a => a.Country)
+                .FirstOrDefault(p => p.SurveyPlanNumber == plan && p.Pin == null && p.Pid == null) ?? throw new KeyNotFoundException();
```

Comment:
We're pretty confident on this I take it? In the long run we may want to back this up with some documentation from LTSA - on both issues: 

1. there is only one non-pid/pin common property for plan number
2. common property can be identified as a parcel with no pid/pin but with a plan (and ends with an 'S')

Reply:
Yes, AFAIK, that is the way we would know if it's the same property. On your second point, we need to have that on the front end too, but I am unsure if its a hotfix or something we can add later.

Reply:
To me it seems like a 5.10 change.